test/src/org/labkey/test/pages/cds/GroupDetailsPage.java

```diff
@@ -0,0 +1,53 @@
+package org.labkey.test.pages.cds;
+
+import org.labkey.test.Locator;
+import org.labkey.test.pages.LabKeyPage;
+import org.labkey.test.util.Ext4Helper;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class GroupDetailsPage extends LabKeyPage<LabKeyPage.ElementCache>
+{
+    public GroupDetailsPage(WebDriver driver)
+    {
+        super(driver);
+        waitForPage();
+    }
+
+    @Override
+    public void waitForPage()
+    {
+        _test.waitForText("Edit details");
```

Comment:
`_test` is deprecated, don't use it. It's especially unnecessary here; `waitForText` is from `WebDriverWrapper`, which `LabKeyPage` extends.
```suggestion
       waitForText("Edit details");
```